packages/app/cypress/e2e/Home.cy.ts

```diff
@@ -20,4 +20,10 @@ describe('Home', () => {
     cy.visit('/wallet');
     cy.contains(/you don't have any assets/i);
   });
+
+  it('should open the side bar and close it', () => {
+    cy.visit('/wallet');
+    cy.get('[data-testid="drawer_trigger"]').eq(1).click();
```

Comment:
We don't use data-testId normally, we prefer to add aria-labels and query by it. It creates better accessibility at the same time, helps with the tests.

Also, to check if the drawer is open, you could query by some menu item text, and after it is closed, query again to see if it is not visible.